test/parallel/test-fs-open-numeric-flags-posix.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+var common = require('../common');
```

Comment:
Please use `const` for import statements.  I think the linter should have complained about that.